site/content/en/docs/commands/podman-env.md

```diff
@@ -4,6 +4,14 @@ description: >
   Configure environment to use minikube's Podman service
 ---
 
+## Requirements
+
+- **Podman version 4.9.2 or newer is required.**
+- Support for Podman v3 and varlink-based communication has been removed. The `podman-env` command now configures your environment to use the Podman REST API socket, as required by Podman v4+.
```

Comment:
I don't think that varlink is relevant here, as it was removed already in Podman v2:

https://podman.io/blogs/2020/06/29/podman-v2-announce.html